wagtail/admin/views/pages/copy.py

```diff
@@ -51,16 +51,24 @@ def copy(request, page_id):
             keep_live = can_publish and form.cleaned_data.get('publish_copies')
 
             # Copy the page
-            new_page = page.specific.copy(
-                recursive=form.cleaned_data.get('copy_subpages'),
-                to=parent_page,
-                update_attrs={
-                    'title': form.cleaned_data['new_title'],
-                    'slug': form.cleaned_data['new_slug'],
-                },
-                keep_live=keep_live,
-                user=request.user,
-            )
+            if can_publish and form.cleaned_data['alias']:
```

Comment:
It took me a while to make sense of the `can_publish` here - I think it would be helpful to add a comment here and on the form definition, along the lines of: "only users with publish permission are given the option of creating an alias, because aliases must keep the same published state as the original".